django/db/migrations/operations/fields.py

```diff
@@ -283,7 +281,7 @@ def state_forwards(self, app_label, state):
                     [self.new_name if n == self.old_name else n for n in together]
                     for together in options[option]
                 ]
-        state.reload_model(app_label, self.model_name_lower, delay=delay)
+        state.reload_model(app_label, self.model_name_lower)
```

Comment:
As I was trying yo point out on the ticket, I'm confused on why this is an issue at all. When using `sqlmigrate` Django produces the correct SQL. However, when running `migrate` Django seems to be doing the wrong thing. I'm think it would be worth investigating in that direction a bit further to see why and where those differences come from, before removing the delayed rendering (entirely)